sample/src/NimblePros.SampleToDo.Infrastructure/Data/Config/VogenEfCoreConverters.cs

```diff
@@ -9,6 +9,7 @@
 namespace NimblePros.SampleToDo.Infrastructure.Data.Config;
 
 [EfCoreConverter<ProjectName>] // not needed for structs
+[EfCoreConverter<ToDoItemId>] // not needed for structs
```

Comment:
Need to specify each type, which is a bit of a pain. But doing this also generates the following code, which means we can skip all of the `HasVogenConversion`:

```c#
namespace NimblePros.SampleToDo.Infrastructure.Data.Config;
    
  internal static class VogenEfCoreConverters__Ext
  {
      internal static global::Microsoft.EntityFrameworkCore.ModelConfigurationBuilder RegisterAllInVogenEfCoreConverters(this global::Microsoft.EntityFrameworkCore.ModelConfigurationBuilder configurationBuilder)
      {
        configurationBuilder.Properties<NimblePros.SampleToDo.Core.ProjectAggregate.ProjectName>().HaveConversion<NimblePros.SampleToDo.Infrastructure.Data.Config.VogenEfCoreConverters.ProjectNameEfCoreValueConverter, NimblePros.SampleToDo.Infrastructure.Data.Config.VogenEfCoreConverters.ProjectNameEfCoreValueComparer>();
configurationBuilder.Properties<NimblePros.SampleToDo.Core.ProjectAggregate.ToDoItemId>().HaveConversion<NimblePros.SampleToDo.Infrastructure.Data.Config.VogenEfCoreConverters.ToDoItemIdEfCoreValueConverter, NimblePros.SampleToDo.Infrastructure.Data.Config.VogenEfCoreConverters.ToDoItemIdEfCoreValueComparer>();


        return configurationBuilder; 
      }
  }
```

Reply:
Ok so if I get rid of the EF Core stuff in Core and add the EfCoreConverter<T> here I should be good. Thanks!